### ISSUE TITLE
[White Label -Card Wizard]: Narrator announcing incorrect role as toggle button for Submit button

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5719]
### What is the PWD impact?

Screen reader users rely on role announcements to understand how a control behaves. When these action buttons are incorrectly announced as toggle buttons, users may think the control needs to be turned on or off, leading to confusion and incorrect interaction expectations.

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: chro and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Card Wizard word in type your message text box.
4. Response will appear.
5. Navigate to the Name control and activate it.
6. Navigate to Address control and activate it.
7. Navigate to Phone email control and activate it.
8. Navigate to submit control.
9. Observe an issue that Narrator announcing incorrect role as toggle button for Positive, Default and Destructive controls.

### What do you expect?

The element should expose a correct programmatic role that reflects its actual behaviour. Submit button should be announced as regular buttons or appropriate action controls, not as toggle buttons.

### What actually happened?

Narrator announcing incorrect role as toggle button for Submit button

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

### Adaptive Card JSON

```json

```
